nixos/modules/security/pam.nix

```diff
@@ -93,6 +94,19 @@ let
         '';
       };
 
+      pamExec = mkOption {
```

Comment:
`pamExec` -> `sessionCommands` or something like that. `pam` is superfluous because we're already in the PAM options, and `exec` is vague.